File: frontend/src/locales/de.js
// German UI messages. Keys shared with src/bundesrag/locales/de.py keep the
// same name there so drift between the two catalogs is easy to grep for.
export default {
  answer_submit: 'Antworten',
  ask_download_count:
    '{num_matched} Dokumente gefunden, davon {num_existing} bereits heruntergeladen. ' +
    'Wie viele der verbleibenden {num_to_download} sollen heruntergeladen werden ' +
    '(die neuesten zuerst, 0 zum Abbrechen)?',
  ask_stats_info:
    'Antwort basierend auf {num_documents} indexierten Dokumenten mit {num_chunks} ' +
    'Textabschnitten, unter Auswahl der {top_k} passendsten Textabschnitte.',
  ask_stats_info_filtered:
    'Antwort basierend auf {num_filtered_documents} von {num_documents} indexierten ' +
    'Dokumenten mit {num_filtered_chunks} von {num_chunks} Textabschnitten (Filter aktiv), ' +
    'unter Auswahl der {top_k} passendsten Textabschnitte.',
  ask_stats_hint:
    '{num_documents} Dokumente mit {num_chunks} Textabschnitten indexiert. ' +
    'Ausgewählt werden die {top_k} passendsten Textabschnitte.',
  ask_stats_hint_filtered:
    'Filter aktiv: {num_filtered_documents} von {num_documents} Dokumenten und ' +
    '{num_filtered_chunks} von {num_chunks} Textabschnitten entsprechen dem Filter. ' +
    'Ausgewählt werden die {top_k} passendsten Textabschnitte.',
  ask_filter_datum_end: 'Datum bis',
  ask_filter_datum_start: 'Datum von',
  ask_filter_kind: 'Dokumentart',
  ask_filter_kind_all: 'Alle',
  ask_filter_wahlperiode: 'Wahlperiode',
  ask_filters_summary: 'Filter (optional)',
  ask_placeholder: 'Welche Gesetzesvorhaben gibt es bzgl. künstlicher Intelligenz?',
  ask_question_label: 'Frage zu den indexierten Dokumenten',
  ask_submit: 'Frage stellen',
  ask_title: 'Fragen',
  cancel_requested: 'Abbruch angefordert …',
  cancel_submit: 'Abbrechen',
  clear_description:
    'Löscht alle heruntergeladenen PDFs, setzt die Vektordatenbank zurück und leert die ' +
    'Liste wartender Dokumente. Dieser Schritt kann nicht rückgängig gemacht werden.',
  clear_submit: 'Alles löschen',
  clear_title: 'Löschen',
  confirm_delete_all: 'Wirklich alle heruntergeladenen Dokumente und die Vektordatenbank löschen?',
  confirm_delete_file:
    'Datei „{file}“ wirklich löschen (inklusive ihrer Einträge in der Vektordatenbank)?',
  confirm_use_query: 'Abfrage so verwenden?',
  count_submit: 'Bestätigen',
  delete_done: 'Fertig: {num_files} Dateien gelöscht und Vektordatenbank zurückgesetzt.',
  delete_file_label: 'Datei löschen',
  download_done: 'Fertig: {num_documents} Dokumente heruntergeladen.',
  download_partial_failure:
    'Achtung: {num_failed} Dokument(e) konnten nicht heruntergeladen werden ' +
    'und wurden übersprungen.',
  download_prompt_label: 'Beschreibung der gewünschten Dokumente',
  download_prompt_placeholder: 'Plenarprotokolle der 21. Wahlperiode.',
  download_running: 'Verarbeitung läuft …',
  download_skipped_existing:
    'Hinweis: {num_skipped} Dokument(e) waren bereits heruntergeladen und wurden übersprungen.',
  download_submit: 'Abfrage erstellen',
  download_title: 'Herunterladen',
  error_prefix: 'Fehler: {error}',
  index_counts: '{num_to_index} Dokument(e) zu indexieren, {num_indexed} bereits indexiert.',
  index_description:
    'Zerlegt alle heruntergeladenen, noch nicht indexierten Dokumente in Textabschnitte und ' +
    'speichert sie in der Vektordatenbank. Das kann je nach Umfang einige Zeit dauern.',
  index_done: 'Fertig: {num_documents} Dokumente, {num_chunks} Textabschnitte gespeichert.',
  index_running: 'Indexierung läuft …',
  index_submit: 'Indexieren starten',
  index_title: 'Indexieren',
  kind_drucksache: 'Drucksache',
  kind_plenarprotokoll: 'Plenarprotokoll',
  nav_ask: 'Fragen',
  nav_clear: 'Löschen',
  nav_download: 'Herunterladen',
  nav_index: 'Indexieren',
  nav_status: 'Status',
  no: 'Nein',
  operation_cancelled: 'Vorgang abgebrochen.',
  progress_count: '{current} von {total}',
  source_show_text: 'Textauszug anzeigen',
  sources_header: 'Quellen:',
  status_file_indexed: 'indexiert',
  status_file_not_indexed: 'nicht indexiert',
  status_chunk_mismatch:
    'Warnung: Die Vektordatenbank enthält {num_chunks} Textabschnitte, laut den ' +
    'Dokument-Metadaten erwartet werden {num_expected} — z. B. durch einen ' +
    'abgebrochenen Indexierungslauf oder außerhalb des Tools geänderte Daten.',
  status_loading: 'Lade Status …',
  status_no_documents: 'Keine Dokumente heruntergeladen.',
  status_num_chunks: 'Textabschnitte in der Vektordatenbank: {count}',
  status_num_downloaded: 'Heruntergeladen: {count}',
  status_num_indexed: 'Indexiert: {count}',
  status_pdf_size: 'Speicherplatz PDFs: {size}',
  status_source_link: 'PDF',
  status_th_actions: 'Aktionen',
  status_th_chunks: 'Textabschnitte',
  status_th_datum: 'Datum',
  status_th_doc_id: 'DIP-ID',
  status_th_dokumentnummer: 'Dokumentnummer',
  status_th_file: 'Datei',
  status_th_kind: 'Art',
  status_th_pages: 'Seiten',
  status_th_source: 'Quelle',
  status_th_status: 'Status',
  status_th_title: 'Titel',
  status_title: 'Status',
  status_vectorstore_size: 'Speicherplatz Vektordatenbank: {size}',
  usage_chat: 'Chat: {input_tokens} Eingabe- + {output_tokens} Ausgabe-Tokens ({num_calls} Aufruf(e))',
  usage_cost: 'Geschätzte Kosten: {cost} {currency}',
  usage_cost_suffix: ', ≈ {cost} {currency}',
  usage_embedding: 'Embeddings: {tokens} Tokens ({num_calls} Aufruf(e))',
  usage_header: 'Mistral-API-Nutzung:',
  usage_op_ask: 'Fragen',
  usage_op_download: 'Herunterladen',
  usage_op_index: 'Indexieren',
  usage_time: 'API-Zeit: {seconds} s',
  usage_totals_header: 'Mistral-API-Nutzung insgesamt:',
  usage_totals_line: '{operation}: {tokens} Tokens (Vorgänge: {num_operations}, API-Zeit: {seconds} s)',
  yes: 'Ja',
}

File: src/bundesrag/locales/de.py
MESSAGES = {
    "answer_must_be_integer": "Die Antwort muss eine ganze Zahl sein.",
    "app_help": "Lade Bundestagsdokumente herunter und stelle Fragen dazu.",
    "ask_help": "Beantwortet QUESTION auf Basis der gespeicherten Dokumente.",
    "ask_query_feedback": "Was soll an der Abfrage angepasst werden?",
    "clear_help": "Löscht alle heruntergeladenen Dokumente und setzt die Vektordatenbank zurück.",
    "clear_yes_option_help": "Ohne Rückfrage löschen.",
    "ask_download_count": (
        "{num_matched} Dokumente gefunden, davon {num_existing} bereits heruntergeladen. "
        "Wie viele der verbleibenden {num_to_download} sollen heruntergeladen werden "
        "(die neuesten zuerst, Enter für alle, 0 zum Abbrechen)? "
    ),
    "confirm_delete_all_yn": (
        "Wirklich alle heruntergeladenen Dokumente und die Vektordatenbank löschen? [j/N] "
    ),
    "confirm_use_query_yn": "Abfrage so verwenden? [j/N] ",
    "confirmation_required": "Bestätigung erforderlich.",
    "delete_done": "Fertig: {num_files} Dateien gelöscht und Vektordatenbank zurückgesetzt.",
    "document_reference": "Drucksache/Protokoll {dokumentnummer}",
    "download_aborted": "Abgebrochen: Download von {count} Dokumenten abgebrochen.",
    "download_done": "Fertig: {num_documents} Dokumente heruntergeladen.",
    "download_help": (
        "Lädt Dokumente passend zu PROMPT herunter, ohne sie zu indexieren.\n"
        "\n"
        "PROMPT ist eine natürlichsprachliche Beschreibung der gesuchten Dokumente. "
        "Ein LLM übersetzt sie in DIP-API-Filter und stellt eine Rückfrage, wenn "
        "die Anfrage zu vage ist.\n"
        "\n"
        "Verfügbare Endpunkte:\n"
        "\n"
        "\b\n"
        "- drucksache: Anträge, Gesetzentwürfe, Kleine Anfragen usw.\n"
        "- plenarprotokoll: Plenarsitzungsprotokolle.\n"
        "\n"
        "Verfügbare Filter (alle optional):\n"
        "\n"
        "\b\n"
        '- datum_start / datum_end: Datumsbereich (z. B. "seit 01.01.2026")\n'
        "- wahlperiode: Wahlperiodennummer (z. B. 21)\n"
        '- dokumentnummer: exakte Drucksachen-/Protokollnummer (z. B. "19/1")\n'
        "- zuordnung: Herkunft — BT (Bundestag), BR (Bundesrat), BV\n"
        "  (Bundesversammlung), EK (Enquete-Kommission); ohne diesen Filter\n"
        "  liefert plenarprotokoll BT- und BR-Protokolle gemischt\n"
        '- drucksachetyp: Dokumenttyp, z. B. "Antrag", "Gesetzentwurf" (nur drucksache)\n'
        '- urheber: Urheber/Fraktion, z. B. "Fraktion der SPD" (nur drucksache)\n'
        "- ressort_fdf: federführendes Bundesministerium (nur drucksache)\n"
        "- titel: Suchbegriffe im Titel, ODER-verknüpft (nur drucksache)\n"
        "\n"
        "Hinweis: urheber und ressort_fdf werden über mehrere Werte UND-verknüpft. "
        "Um Dokumente von einem von zwei Urhebern zu finden, sind zwei getrennte "
        "download-Aufrufe nötig."
    ),
    "download_partial_failure": (
        "Achtung: {num_failed} Dokument(e) konnten nicht heruntergeladen werden "
        "und wurden übersprungen."
    ),
    "download_skipped_existing": (
        "Hinweis: {num_skipped} Dokument(e) waren bereits heruntergeladen und wurden übersprungen."
    ),
    "file_not_found": "Datei nicht gefunden.",
    "filter_dokumentnummer": "  Dokumentnummer: {value}",
    "filter_drucksachetyp": "  Drucksachetyp: {value}",
    "filter_ressort": "  Ressort: {value}",
    "filter_titel": "  Titel enthält: {value}",
    "filter_urheber": "  Urheber: {value}",
    "filter_wahlperiode": "  Wahlperiode: {value}",
    "filter_zeitraum": "  Zeitraum: {start} bis {end}",
    "filter_zuordnung": "  Zuordnung: {value}",
    "index_counts": ("{num_to_index} Dokument(e) zu indexieren, {num_indexed} bereits indexiert."),
    "index_done": "Fertig: {num_documents} Dokumente, {num_chunks} Textabschnitte gespeichert.",
    "index_help": "Indexiert zuvor heruntergeladene, aber noch nicht indexierte Dokumente.",
    "job_not_cancellable": "Der Job läuft nicht mehr und kann nicht abgebrochen werden.",
    "job_not_found": "Job nicht gefunden.",
    "job_not_waiting": "Der Job wartet nicht auf eine Eingabe.",
    "operation_cancelled": "Vorgang abgebrochen.",
    "page_short": "S. {page}",
    "page_suffix": ", Seite {page}",
    "planned_query_header": "Geplante DIP-Abfrage (Endpunkt: {endpoint}):",
    "progress_step": "[Schritt {n}/{total}] {name}",
    "query_agent_failed": (
        "Konnte aus der Anfrage auch nach mehreren Rückfragen keine gültige DIP-Abfrage erstellen."
    ),
    "serve_help": "Startet die lokale Weboberfläche (FastAPI + Vue SPA).",
    "serve_host_option_help": "Adresse, an die der Webserver gebunden wird.",
    "serve_port_option_help": "Port des Webservers.",
    "serve_reload_option_help": (
        "Bei Quellcode-Änderungen automatisch neu laden (nur für die Entwicklung)."
    ),
    "serve_started": "Weboberfläche läuft auf http://{host}:{port}/ (Beenden mit Strg+C).",
    "similarity_suffix": " (Ähnlichkeit: {score})",
    "sources_header": "\nQuellen:",
    "status_file_indexed": "indexiert",
    "status_file_not_indexed": "nicht indexiert",
    "status_chunk_mismatch": (
        "Warnung: Die Vektordatenbank enthält {num_chunks} Textabschnitte, laut den "
        "Dokument-Metadaten erwartet werden {num_expected} — z. B. durch einen "
        "abgebrochenen Indexierungslauf oder außerhalb des Tools geänderte Daten."
    ),
    "status_files_header": "Dateien:",
    "status_help": "Zeigt an, wie viele Dokumente heruntergeladen und indexiert sind.",
    "status_num_chunks": "Textabschnitte in der Vektordatenbank: {count}",
    "status_num_downloaded": "Heruntergeladen: {count}",
    "status_num_indexed": "Indexiert: {count}",
    "status_pdf_size": "Speicherplatz PDFs: {size}",
    "status_vectorstore_size": "Speicherplatz Vektordatenbank: {size}",
    "step_download_pdfs": "PDFs herunterladen",
    "step_generate_answer": "Antwort erzeugen",
    "step_interpret_request": "Anfrage interpretieren",
    "step_search_documents": "Dokumente suchen",
    "step_search_passages": "Passagen suchen",
    "unexpected_error": "Ein unerwarteter Fehler ist aufgetreten. Details siehe Logdatei.",
    "unknown_document": "Unbekanntes Dokument",
    "usage_chat": (
        "  Chat: {input_tokens} Eingabe- + {output_tokens} Ausgabe-Tokens ({num_calls} Aufruf(e))"
    ),
    "usage_cost": "  Geschätzte Kosten: {cost} {currency}",
    "usage_cost_suffix": ", ≈ {cost} {currency}",
    "usage_embedding": "  Embeddings: {tokens} Tokens ({num_calls} Aufruf(e))",
    "usage_header": "Mistral-API-Nutzung:",
    "usage_op_ask": "Fragen",
    "usage_op_download": "Herunterladen",
    "usage_op_index": "Indexieren",
    "usage_time": "  API-Zeit: {seconds} s",
    "usage_totals_header": "Mistral-API-Nutzung insgesamt:",
    "usage_totals_line": (
        "  {operation}: {tokens} Tokens (Vorgänge: {num_operations}, API-Zeit: {seconds} s)"
    ),
}
